src/components/Form/FormSpinBox.tsx

```diff
@@ -30,11 +30,20 @@ export function FormSpinBox<T>({
   errors,
   touched,
 }: FormSpinBoxProps<T>) {
-  const isTouched = _.get(touched, identifier)
```

Comment:
Not really a fan of isTouched here, since it only throws errors after your mouse clicks out of the input. (To me it's better to get the error realtime.) I'm not a front end dev though, is there a reason why we may want this behavior?

Reply:
@dannysepler Hi Danny,  thanks so much for the contribution.

I am not a fan of `isTouched` either, but it is how author of Formik recommends to do and there was a huge figth sometimes ago on Twitter and it was established that it is the best practice. Not that our app follows any UX best practices. That's probably the only one :)

But more importantly, in our particular app, removing it makes the typing in the field quite laggy, as we now run validation on every keystroke, and, thus, dispatch `useReducer` actions. Which is a very dumb thing to do in the validation in the first place. Hope we can find a way to fix it soon. See more about it here: https://github.com/neherlab/covid19_scenarios/issues/17

Does your fix still works with `isTouched`? If yes, could you please bring it back for now?

Reply:
ha, it'll break my heart but can do!

Reply:
Yes, a part of me died too ;)